packages/notifi-react-card/lib/assets/PenIcon.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+export type Props = Readonly<{
+  className?: string;
+}>;
+
+export const PenIcon: React.FC<Props> = ({ className }: Props) => {
```

Comment:
should we update this to take svg props too?

Reply:
👍